src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -46,6 +47,18 @@ namespace big
 		g_player_service->player_join(player);
 		if (net_player_data)
 		{
+			auto bplyr = g_player_service->get_by_id(net_player_data->m_gamer_handle.m_rockstar_id);
```

Comment:
You need to use `player->m_player_id` for this function.
Use `g_player_service->get_by_rid` for RID.